app/components/polls/questions/answers_component.rb

```diff
@@ -10,16 +10,18 @@ def already_answered?(answer)
     user_answers.find_by(answer: answer.title).present?
   end
 
-  def voted_before_sign_in?
-    user_answers.any? do |vote|
-      vote.updated_at < current_user.current_sign_in_at
-    end
-  end
-
   def question_answers
     question.question_answers
   end
 
+  def user_answer(question_answer)
```

Comment:
I see the method `already_answered?` receives a parameter called `answer` but the `user_answer` and `disable_answer?` methods receive a parameter called `question_answer`. It took me a while to realize that in all cases the received parameter was the same object :sweat_smile:; what do you think about using the same parameter name for all these methods? Either name is fine with me.